### ISSUE TITLE
Any example or document for promise?

### ISSUE THREAD

**Issue by sj82516** [reasonml/reasonml.github.io#301]
Hi there, 
I am JS user and I heavily use promise combined with async/await syntax a lot.  
But I didn't find out any example or document about Promise?  
Would you please add some example about promise such as setTimeout/Promise/Promise.all([])/make http request and so forth ?   
It would be super useful. 
Thanks.

**Comment by jaredly** [comment]
I added some docs, but in the wrong place :P here's the PR https://github.com/reasonml/reasonml.github.io/pull/286

**Comment by sj82516** [comment]
cool. :) 
Thanks a lot

**Comment by chenglou** [comment]
Yep, this has since been fixed here: https://reasonml.github.io/docs/en/promise.html

**Comment by chenglou** [comment]
Keeping this open until async/await

**Comment by yawaramin** [comment]
Relevant Reason issue is https://github.com/facebook/reason/issues/1321
